src/pages/PublicModule/PublicModule.tsx

```diff
@@ -106,6 +107,7 @@ class PublicModule extends Component<PublicModuleProps, MyModuleState> {
         loading={this.state.startingModule}
         className='btn btn-primary'
         onClick={this.hasUserModule() ? undefined : this.startModule}
+        toolTipText={'This lab is currently disabled'}
```

Comment:
This tooltip text only shows when the button is disabled but there is no logic here to disable the button